Transfer/Sources/CreateTransferController.swift

```diff
@@ -47,13 +47,19 @@ public final class CreateTransferController: UITableViewController {
         navigationItem.backBarButtonItem = UIBarButtonItem.back
         largeTitle()
         setViewBackgroundColor()
+        initializePresenter()
+        presenter.loadCreateTransfer()
+        setUpCreateTransferTableView()
+        hideKeyboardWhenTappedAround()
+    }
+
+    private func initializePresenter() {
         if let clientTransferId = initializationData?[InitializationDataField.clientTransferId] as? String {
             let sourceToken = initializationData?[InitializationDataField.sourceToken] as? String
             presenter = CreateTransferPresenter(clientTransferId, sourceToken, view: self)
+        } else {
+            fatalError("Required data not provided in initializePresenter")
```

Comment:
minor : Maybe better to add " for create transfer" here - so that the error messages are specific to the controller